Examples-Web/AnimatedProgressButtons/LinearProgress/LinearProgress.js

```diff
@@ -0,0 +1,32 @@
+window.onload = function () {
+    TouchFree.Init({ initialiseCursor: false });
+    AddTouchFreeCursor();
+    TouchFree.RegisterEventCallback("TransmitInputAction", AnimateLinearProgress);
+};
+
+// Animates a clip path for active overlay elements, identified by the "hovered" id
+function AnimateLinearProgress(inputAction) {
+    const hoveredElement = document.querySelector(".hovered");
+    if (hoveredElement) {
+        const progressPercent = inputAction.ProgressToClick * 100;
+        document.querySelector(":root").style.setProperty("--progress", `${progressPercent}%`);
+    }
+}
+
+function AddTouchFreeCursor() {
```

Comment:
💬 Do we want to use the new Init cursor option? Or leave this as an example of how to add a custom cursor?
(I think we should have specific cursor examples in the future)

Reply:
I think this is probably best left as it's a good cursor for this example (has no ring and doesn't resize) leaving the gradient to show progress by itself. I agree that we should create some specific cursor examples though :+1: